CacheManager.StackExchange.Redis.SelfCleaning.Examples/ManualScenariosRunner.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Channels;
+using CacheManager.Core;
+using CacheManager.Redis;
+using CacheManager.StackExchange.Redis.SelfCleaning.Core;
+using CacheManager.StackExchange.Redis.SelfCleaning.Examples;
+using CacheManager.StackExchange.Redis.SelfCleaning.Examples.Scenarios;
+using StackExchange.Redis;
+using static CacheManager.StackExchange.Redis.SelfCleaning.Examples.Utilities;
+
+namespace CacheManager.StackExchange.Redis.SelfCleaning.Examples
+{
+    public class ManualScenariosRunner
+    {
+        private TimeSpan _cleanupInterval = TimeSpan.FromSeconds(0.1);
+        private TimeSpan _slidingExpiration = TimeSpan.FromSeconds(1);
+        private string _connectionString = "localhost:6379";
+
+        public void Run()
+        {
+            Console.WriteLine("Hello to you dear monsieur/mademoiselle, welcome to our Reidis-Mania! Lets run some weird scenarios!");
+            GetParametersFromUser();
+
+            RunConfiguredScenarios();
+            
+            Console.WriteLine("\n\n\n Done running scenarios, press any key to exit. See you next time!");
+            Console.Read();
+        }
+
+        private void RunConfiguredScenarios()
```

Comment:
f, except of CreateCacheManager function which I leave at the end because it is being transfered to the Secnarios as Action and isnt being called, ok?

Reply:
👍